### DISCUSSION TITLE
Annoying message when connecting to a SSH server with X11 Forwarding: `X11 connection rejected because of wrong authentication`

### DISCUSSION THREAD

**Discussion by maicol07** [microsoft/wslg#571] [Q&A]
Hi,
I'm connecting via `ssh -X` to my server from WSL2, but I get this annoying message every second:
```
X11 connection rejected because of wrong authentication.
```

Do you know how to fix it?
Thanks

Edit: It seems that there is another warning when starting the SSH session:
```
Warning: No xauth data; using fake authentication data for X11 forwarding.
```

**Comment by Biswa96** [comment]
You have to enable X11Forwarding in sshd_config and install xauth (or xorg-xauth) package.

**Reply by maicol07** [reply]
I've already had enabled `X11Forwarding` and installed the xauth package.

**Comment by dibalavs** [comment]
Hello. I have the same issue. It looks like wslg does not set XAUTHORITY environment variable
Is it bug in wslg?
